translations/strings.yaml

```diff
@@ -1,288 +1,297 @@
+customDNSSettings:
+  dnsInfoAlert: VPN must be off to change these settings
+  dnsNavItem: DNS settings
+  dnsDefaultToggle: Default
+  dnsDefaultRadioHeader: Use default DNS
+  dnsDefaultRadioBody: Automatically use Mozilla VPN-protected DNS
+  dnsAdvancedToggle: Advanced
+  dnsAdblockRadioHeader: Use adblock DNS
+  dnsAdblockRadioBody: Block internet ads with VPN
+  dnsAntitrackRadioHeader: Use anti-tracking DNS
+  dnsAntitrackRadioBody: Block harmful domains with VPN
+  dnsAdblockAntiTrackRadioHeader: Use adblock and anti-tracking DNS
+  dnsAdblockAntiTrackRadioBody: Block internet ads and harmful domains with VPN
+  dnsCustomDNSRadioHeader: Use custom DNS
+  dnsCustomDNSRadioBody: Resolve websites using a custom DNS
+  dnsFieldPlaceHolder: Enter custom DNS
+  # The below is an inline error when an invalid DNS is entered into the custom DNS field
+  inlineCustomDNSError: Invalid DNS, please try a new one
+
+networkSettings:
+  multihopTitle: Multi-hop tunnel
+
+notifications:
+  captivePortalBlockTitle: Guest Wi-Fi portal blocked
+  captivePortalBlockMessage: The guest Wi-Fi network you’re connected to requires action. Click here to turn off VPN to see the portal.
+  captivePortalUnblockTitle: Guest Wi-Fi portal detected
+  captivePortalUnblockMessage: The guest Wi-Fi network you’re connected to may not be secure. Click here to turn on VPN to secure your device.
+  unsecuredNetworkTitle: Unsecured Wi-Fi network detected
+  unsecuredNetworkMessage:
+    value: "%1 is not secure. Click here to turn on VPN and secure your device."
+    comment: "%1 is the Wi-Fi network name"
+
+systray:
+  hide: Hide Mozilla VPN
+  show: Show Mozilla VPN
+  statusConnectedTo: "Connected to:"
+  statusConnectTo: "Connect to the last location:"
+  statusConnectingTo: "Connecting to:"
+  statusDisconnectingFrom: "Disconnecting from:"
+  # location2 to avoid conflicts with other branches
+  location2:
+    value: "%1, %2"
+    comment: Location in the systray. %1 is the country, %2 is the city.
+  disconnect: Disconnect
+  help: Help
+  preferences: Preferences…
+  quit: Quit Mozilla VPN
+
+settings:
+  systemPreferences: System preferences
+  startAtBootDescription: Mozilla VPN will launch when you open the app
+  dataCollectionDescription: Allow Mozilla VPN to send technical, interaction, and campaign and referral data to Mozilla
+
+aboutUs:
+  licenses: Licenses
+
+AndroidNotifications:
+  GeneralNotifications:
+    value: General Notifications
+    comment: A notification channel; so the user can subscribe to "General Notifications"
+
+updateButton:
+  checkForUpdateButtonText: Check for Updates
+  titleNoUpdate: You are up to date
+  descriptionNoUpdate: There is no new update available at this time
+  actionNoUpdate: Okay
+  titleOnUpdate: Updates available
+  descriptionOnUpdate: Click to update to the latest version of the application

```

Comment:
@chrisrooneys 
Noticed this while translating. Does this show up only on desktop clients? If not, "click" might not be the best choice.